projects/plugins/crm/includes/ZeroBSCRM.Core.Menus.Top.php

```diff
@@ -269,6 +269,8 @@ function zeroBSCRM_admin_top_menu( $branding = 'zero-bs-crm', $page = 'dash' ) {
 			});
 		})
 		</script>
+		<style>
+		</style>
```

Comment:
Do we need this empty style element?

Reply:
Leftover from tests! Thanks for pointing that out!